service/src/main/java/uk/gov/hmcts/reform/fpl/enums/DocmosisTemplates.java

```diff
@@ -3,7 +3,8 @@
 public enum DocmosisTemplates {
     C6("FL-PLW-HNO-ENG-00092.docx", "Notice_of_proceedings_c6.pdf"),
     C6A("FL-PLW-HNO-ENG-00093.docx", "Notice_of_proceedings_c6a.pdf"),
-    SDO("FL-PLW-STD-ENG-00099.doc", "standard_directions_order.pdf");
+    SDO("FL-PLW-STD-ENG-00099.doc", "standard_directions_order.pdf"),
+    C21("FL-PLW-GOR-ENG-00218.docx","C21_Order");
```

Comment:
Can the c21 not also include the extension so we do not need to do C21.getDocumentTitle() + ".pdf"?

Reply:
I want to append a number onto the order filename (e.g C21_Order_1.pdf) so that it's easier for the users to distinguish between the links. They can create multiple orders - we don't want all orders to have the same link name as this looks weird